### ISSUE TITLE
Inconsistent color of some mimetypes

### ISSUE THREAD

**Issue by hlechner** [numixproject/numix-icon-theme#959]
The following colors could be found on mimetypes of 64px that uses ``dcdcdc`` as background:
``#2d2d2d`` - x-tex
``#3e3e3e`` - x-chdr, x-c++hdr

``#515151`` - ascii, font, octet-stream, script-blank
``#555555`` - x-lilypond

``#878787`` - document, x-cd-image, x-desktop, x-trash, x-makefile, x-changelog, x-copying, xml, x-readme, x-script, unknown, x-ms-dos-executable, executable, x-riff
``#888888`` - x-osm+xml, encrypted, encrypted-key, text-x-vcard, amiga-disk-format

**Further Information**: colors: ``#878787`` and ``#515151`` seems that are used by mistake because the 48px of the same mimes uses the ``#888888`` and ``#555555`` Instead.

I think could be nice to use only two colors: ``#555555`` and ``#888888`` (or just one) using this conversion:

``#2d2d2d`` -> ``#555555``
``#3e3e3e`` -> ``#555555``
``#515151`` -> ``#555555``
``#878787`` -> ``#888888``

**Comment by hlechner** [comment]
```
ascii - 								      32 [#545454] / 48 [#555555] / 64 [#515151]
font - 			16 [#545454] / 24 [#545454] / 32 [#545454] / 48 [#555555] / 64 [#515151]
octet-stream - 	16 [#545454] / 24 [#545454] / 32 [#545454] / 48 [#555555] / 64 [#515151]
script-blank -	16 [#545454] / 24 [#545454] / 32 [#545454] / 48 [#555555] / 64 [#515151]




document - 			16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888/#555555] / 64 [#878787]
document-template - 16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888/#555555] / 64 [#878787]
executable - 		16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
text - 				16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
text-template -		16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
cd-image - 			16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
x-desktop -			16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
trash -				16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
x-makefile -		16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
x-changelog -		16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
x-copying - 		16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
xml - 				16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
x-readme -			16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
x-script -			16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
unknown -			16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
x-riff -			16 [#888888] / 24 [#888888] / 32 [#888888] / 48 [#888888] / 64 [#878787]
```

**Comment by wa4557** [comment]
This is fixed, right? (#963)

**Comment by Foggalong** [comment]
I believe so :+1:
